src/main/resources/locales/cy/delete-court-subscription-confirmation.json

```diff
@@ -0,0 +1,10 @@
+{
+    "header": "Are you sure you want to delete all the subscription?",
```

Comment:
Similar to the above, I think subscription should be subscriptions (Same below on line 5)

Reply:
fixed.